### PR TITLE
Doc: add readMe information

### PR DESCRIPTION
**Pull request by LSeward0421**
# Description

- updated ReadMe. **NEED to add deploy link once finished. Currently has placeholder**

## Type of change

- [x]  New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

- [x]  local host
- [x]  dev tools
- [x]  terminal

# Related Issues:

- [x]  closes #12 
- [ ]  and/or related to #<issue number>

# Checklist:

- [x]  My code follows the Turing's guidelines of this project
- [x]  I have performed a self-review of my own code
- [x]  No console error messages

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "skipLibCheck": true,
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
-    "strict": true,
+    "strict": false,
```
Comment (kirkhauck) [inline tsconfig.json:13]: Could we change this back to true? I think it would be best for a TypeScript project.

**Review by kirkhauck — CHANGES_REQUESTED** [review]
Great README! Thank you so much. Let's change strict mode back to true and we should be dandy!

File: tsconfig.json (hunk 1 above)
Comment (LSeward0421) [reply tsconfig.json:13]: good catch! updated to true now :)

**Review by kirkhauck — APPROVED** [review]
Beautiful! We did it!

**Pull request merged**